distribution/pom.xml

```diff
@@ -90,6 +90,8 @@
                                 <argument>io.druid.extensions:postgresql-metadata-storage</argument>
                                 <argument>-c</argument>
                                 <argument>io.druid.extensions:druid-s3-extensions</argument>
+                                <argument>-c</argument>
+                                <argument>io.druid.extensions:druid-variance</argument>
```

Comment:
how do you feel about calling this module druid-stats?
we can extend it in the future with more statistical functions